packages/component-library/stories/HorizontalBarChart.story.js

```diff
@@ -225,4 +225,68 @@ export default () =>
         );
       },
       { notes }
+    )
+    .add(
+      "Example: Stacked bar chart",
+      () => {
+        const sampleStackedData = [
+          { name: "Poodle", x: "2017", y: 7 },
+          { name: "Poodle", x: "2018", y: 2 },
+          { name: "Poodle", x: "2019", y: 3 },
+          { name: "Bulldog", x: "2017", y: 2 },
+          { name: "Bulldog", x: "2018", y: 5 },
+          { name: "Bulldog", x: "2019", y: 4 },
+          { name: "Terrier", x: "2017", y: 1 },
+          { name: "Terrier", x: "2018", y: 2 },
+          { name: "Terrier", x: "2019", y: 3 }
+        ];
+        const dataValue = text("Data value", "x", GROUP_IDS.DATA);
```

Comment:
_(Leaving this note on the story, but this requires changes to the component)_

Since this is the attribute used for the label displayed on the chart, I would have this be the `dataLabel` prop.